modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -107,6 +107,7 @@ export default {
 
 <style lang="scss" scoped>
   .apos-schema /deep/ .apos-field {
-    margin-bottom: 30px;
```

Comment:
leave this be, the spacing was specific to the login form

Reply:
Done